doc/security/use_cases/index.md

```diff
@@ -0,0 +1,185 @@
+---
+title: "OpenTitan Use Cases"
+---
+
+
+## Overview
+
+OpenTitan’s mission is to raise the security bar industry wide by implementing a transparent, logically secure hardware root of trust with wide application.
+
+This document describes some of those use cases for OpenTitan.
+These range from data center integrations to implement platform integrity solutions, to embedded security applications such as security keys and smart cards.
+References to relevant specifications and certification targets are noted where use cases are backed by industry standards.
+
+
+## Platform Integrity Module
+
+An OpenTitan IC used as a Platform Integrity Module interposes between a platform’s boot flash and its main boot devices such as BMC, PCH and CPU.
```

Comment:
* nit: ASCII `'`
* Can you spell out the BMC and PCH acronyms?

Reply:
Done